vscode-wpilib/src/java/java.ts

```diff
@@ -27,8 +26,8 @@ export async function activateJava(context: vscode.ExtensionContext, coreExports
 
   const javaDebugExtension = vscode.extensions.getExtension('vscjava.vscode-java-debug');
   if (javaDebugExtension === undefined) {
-    // TODO: Make this a visible warning message when project detected is java
-    logger.log('Could not find java extension. Debugging is disabled.');
+    vscode.window.showWarningMessage('Could not find java extension. Debugging is disabled.');
```

Comment:
Does this trigger properly? I haven't traced the code, but will this (properly) only trigger when the project detected is Java?